phpunit.xml.dist

```diff
@@ -8,6 +8,7 @@
   <php>
     <ini name="error_reporting" value="32767"/>
     <ini name="memory_limit" value="-1"/>
+    <env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled"/>
```

Comment:
### Context
I think the goal of this is to suppress a deprecation warning caused by a contrib module (group).
Details:
- The warning is emitted in FieldTypePluginManager.php:113, after analyzing a field type definition.
- The field type is defined in 'group' module, class RelationshipEntityReferenceItem. The problem is the `category = @Translation("Reference"),` in the annotation.
- The message is "Using a translatable string as a category for field type is deprecated in drupal:10.2.0 and is removed from drupal:11.0.0." (it does not mention the field type id).
- The issue exists here and already has a patch: https://www.drupal.org/project/group/issues/3458530

### Problem
I am skeptical about this change, because it also suppresses deprecation warnings caused by our own code.

A simple way to test is to create a unit test case like below, and run only that test, with and without the `<env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled"/>` in phpunit.xml.

```php
<?php

declare(strict_types=1);

namespace Drupal\Tests\collabora_online\Unit;

use Drupal\Tests\UnitTestCase;

class ExampleTest extends UnitTestCase {

    public function testDeprecation() {
        // Optionally add or remove the `@` error suppression.
        @trigger_error('Something is deprecated in ' . __METHOD__, E_USER_DEPRECATED);
        $this->addToAssertionCount(1);
    }

}
```
### Alternative
Apply the patch from https://www.drupal.org/project/group/issues/3458530, or 

### Temporary solution
I think it could be ok to keep this commit in the epic branch, and then fix it before we merge the epic branch.
I don't think it should be merged into the 1.x branch as standalone change.

Reply:
Instead of the current solution, we will apply the patch to disable the deprecation error while testing.
I would not find a way to disable specific paths for deprecations.